### DISCUSSION TITLE
How do you move an app from the leader node to a worker node?

### DISCUSSION THREAD

**Discussion by silasabbott** [caprover/caprover#1978] [Q&A] [answered]
I deployed an app (Plausible) before enabling clusters and attaching a new node. How do I now move Plausible to this new worker node and get it off of my lead node? Important to note that it has persistent storage.

Do I have to start over for it to be deployed on a worker node?

**Comment by githubsaturn** [comment, marked as answer]
Persistent storage means that the data resides on the disk of the node it's deployed to. You can change the node ID in the app config page, and Save & Restart the app. But you'll lose all your data. 

You can manually move the stored data to the new node if you want to.

**Reply by silasabbott** [reply]
Thanks, much appreciated! Thankfully I don't need the persistent data. I'll give it a shot later

**Reply by silasabbott** [reply]
Worked great, thank you! One last thing—on new deployments, how exactly does CapRover determine the node to deploy a service/app to? Is there a way to define which node when creating a new one-click app or deployment?

**Reply by githubsaturn** [reply]
CapRover doesn't do anything! It's all Docker swarm scheduler!

Except if the app is set as a "Persistent app". In that case, CapRover makes sure that the service is locked down to a node.

**Reply by silasabbott** [reply]
Amazing, thanks! Appreciate your time immensely
